Tools-Override/crossgen.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+# Restores crossgen and runs it on all tools components.
+
+__CoreClrVersion=1.1.0
+
+usage()
+{
+    echo "crossgen.sh <directory>"
+    echo "    Restores crossgen and runs it on all assemblies in <directory>."
+    exit 0
+}
+
+restore_crossgen()
+{
+    __pjDir=$__toolsDir/crossgen
+    mkdir -p $__pjDir
+    echo "{\"frameworks\":{\"netcoreapp1.1\":{\"dependencies\":{\"Microsoft.NETCore.Runtime.CoreCLR\":\"$__CoreClrVersion\"}}},\"runtimes\":{\"$__rid\":{}}}" > "$__pjDir/project.json"
+    $__dotnet restore $__pjDir/project.json --packages $__packagesDir
+    cp $__packagesDir/runtime.$__rid.Microsoft.NETCore.Runtime.CoreCLR/$__CoreClrVersion/tools/crossgen $__sharedFxDir
```

Comment:
$__rid is not always going be correct here, since the run-time package and RID may not match (consider rhel.7.2 which will end up pulling down a rhel.7 package or centos.7 which also pulls down a rhel.7 package).  One option may be to restore to some temp dir we haven't put any other packages in an then just pick the only runtime package in that folder.